Cargo.toml

```diff
@@ -1,7 +1,7 @@
 [package]
 name        = "cargo-audit"
 description = "Audit Cargo.lock for crates with security vulnerabilities"
-version     = "0.6.1"
+version     = "0.7.0"
```

Comment:
Don't bump this quite yet. I bump it in the commit where I cut a new release.

Reply:
Reverted.